package.json

```diff
@@ -85,7 +85,7 @@
     "@netlify/framework-info": "9.8.10",
     "@netlify/local-functions-proxy": "1.1.1",
     "@netlify/serverless-functions-api": "1.5.1",
-    "@netlify/zip-it-and-ship-it": "9.13.0",
+    "@netlify/zip-it-and-ship-it": "9.13.1",
```

Comment:
Did you mean to include a ZISI update as part of this?

Reply:
yes, because that contains the updated `@netlify/serverless-functions-api`